simulaqron/tests/auto/quick/network/test_Network.py

```diff
@@ -5,78 +5,74 @@
 from timeit import default_timer as timer
 
 from simulaqron.toolbox import get_simulaqron_path
-from simulaqron.settings import Settings
-from simulaqron.configFiles import construct_node_configs
+from simulaqron.toolbox.manage_nodes import NetworksConfigConstructor
+from simulaqron.settings import simulaqron_settings
 from simulaqron.network import Network
-from simulaqron.general.hostConfig import load_node_names
 
 
 class TestInitNetwork(unittest.TestCase):
     def setUp(self):
         self.network = None
-
-        # Set config files
-        simulaqron_path = get_simulaqron_path.main()
-        nodes_config_file = os.path.join(simulaqron_path, "config", "Nodes.cfg")
-        with open(nodes_config_file, 'w') as f:
-            self.nodes = ["Test1", "Test2", "Test3"]
-            f.writelines([node + "\n" for node in self.nodes])
-        topology_config_file = os.path.join(simulaqron_path, "config", "topology.json")
-        with open(topology_config_file, 'w') as f:
-            self.topology = {"Test1": ["Test2"], "Test2": ["Test3"], "Test3": []}
-            json.dump(self.topology, f)
-        Settings.CONF_TOPOLOGY_FILE = os.path.join("config", "topology.json")
-        # Settings.set_setting("BACKEND", "topology_file", "config/topology.json")
+        self.default_nodes = ["Alice", "Bob", "Charlie", "David", "Eve"]
+        self.default_topology = None
+
+        # # Set config files
```

Comment:
Lots of commented out code. Will it still be used in the future?

Reply:
No, I should delete this